packages/manager/modules/nutanix/src/index.js

```diff
@@ -15,9 +15,6 @@ angular
         resolve: {
           breadcrumb: () => 'nutanix',
         },
-      });
-      $stateProvider.state('nutanix.index.**', {
```

Comment:
May be i am wrong, but if you removed this state provider (`nutanix.index`), i am note sure the redirection will works for option `redirectTo: 'nutanix.index'`, right ?

Reply:
I removed it because it was repeated. We have the same state in `routing.js`